packages/theming/theme-types/src/Theme.types.ts

```diff
@@ -25,7 +25,7 @@ export interface Theme {
   spacing: Spacing;
   host: {
     // appearance of the theme, this corresponds to the react-native Appearance library values, though can be overwritten
-    appearance: 'light' | 'dark';
+    appearance: 'light' | 'dark' | 'auto';
```

Comment:
Should this go into name?: ?

Reply:
While using PlatformColor(), the theme seemed to not use the react-native appearance API at all, so the Light/Dark picker we had did nothing and it just followed the system. 

I think I wanted to add this so that I could then have the picker be disabled if it saw the theme handled it's own changes, but I never added the code to disable the picker/check